caexporter.go

```diff
@@ -140,6 +150,7 @@ func main() {
 	prometheus.MustRegister(newCollector())
 
 	http.Handle("/metrics", promhttp.Handler())
+	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {w.Write([]byte(`ok`))})
```

Comment:
why this needed?

Reply:
for liveness probes in k8s

Reply:
You should not use neither of those imo for exporters. As long as process not crashing exporter should work. In general in my opionon it's a bad habbit to put `liveness` or `readiess` on exporters, since those can affect entire pod if run as sidecar.

I suggest removing this capability.